services/kommander/0.1.0/kommander.yaml

```diff
@@ -21,10 +21,10 @@ spec:
   interval: 15s
   install:
     remediation:
-      retries: 30
+      retries: 0
   upgrade:
     remediation:
-      retries: 30
+      retries: 0
```

Comment:
Upgrades are remiediated via rollback by default (https://fluxcd.io/docs/components/helm/api/#helm.toolkit.fluxcd.io/v2beta1.UpgradeRemediation) 
Do we know this to be (or  consider this) equally harmful?

Reply:
I'm 99% sure a rollback won't work with kommander. Completely untested at this point.